### PR TITLE
Update Changelog to include Docs Only Release

### PR DESCRIPTION
**Pull request by imshashank**
Also removed method to fix endpoints.json file as it's no longer needed.

### CODE REVIEW

**Comment by imshashank** [comment]
@cjyclaire @jeskew Can you please quick review this.

File: build/changelog/ChangelogBuilder.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (cjyclaire) [inline build/changelog/ChangelogBuilder.php]: Could you remove the extra line?

File: tests/Build/Changelog/ChangelogBuilderTest.php [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (cjyclaire) [inline tests/Build/Changelog/ChangelogBuilderTest.php]: Also here?

File: build/changelog/ChangelogBuilder.php [hunk 3]
```diff
@@ -67,9 +67,7 @@ private function cleanJSON($arr)
             if ($x->type == 'NEW_SERVICE') {
```
Comment (jeskew) [inline build/changelog/ChangelogBuilder.php]: This `for` loop isn't really doing anything anymore. You could just set `$this->newServiceFlag` to `count(array_filter($arr, function($change) { return $change->type === 'NEW_SERVICE'; }) > 0`  and return the array that was passed in.

Comment (imshashank) [inline build/changelog/ChangelogBuilder.php]: Good suggestion. Implemented this. Thanks

**Pull request merged**